src/promptflow-devkit/promptflow/_sdk/_orchestrator/test_submitter.py

```diff
@@ -546,24 +551,50 @@ def change_logger_level(level):
             with change_logger_level(level=logging.WARNING):
                 chat_inputs, _ = self.resolve_data(inputs=inputs)
 
+            init_kwargs = None
+            if isinstance(self.flow, Prompty):
+                # Override prompt output format configuration and only return first choice in interactive mode.
+                init_kwargs = {"model": {"parameters": {"stream": False}, "response": "first"}}
```

Comment:
why set this to stream False? I remember we enabled streaming in interactive mode

Reply:
revert it. no need to set stream to false